app/javascript/Snackbar/SnackbarItem.jsx

```diff
@@ -16,7 +16,9 @@ export const snackbarItemProps = {
 
 export const SnackbarItem = ({ message, actions = [] }) => (
   <div className="crayons-snackbar__item flex">
-    <div className="crayons-snackbar__body">{message}</div>
+    <div className="crayons-snackbar__body" role="alert">
```

Comment:
snackbar messages are alerts/notifications for a user, so for a11y purposes, we treat them as an alert